### PR TITLE
Release v1.4

### PR DESCRIPTION
**Pull request by francisco-dlp**

### CODE REVIEW

**Comment by francisco-dlp** [comment]
I suggest Wednesday next week as a deadline to merge any PR that does not fix a bug to leave time to fix any issues discovered during the release process. The release is scheduled for next Friday.

File: setup.py [hunk 1]
```diff
@@ -299,6 +300,7 @@ def __exit__(self, type, value, traceback):
                 'tests/drawing/plot_signal2d/*.png',
                 'tests/drawing/plot_markers/*.png',
                 'tests/drawing/plot_widgets/*.png',
+                'tests/drawing/plot_signal_tools/*.png',
```
Comment (ericpre) [inline setup.py:305]: Sorry, I noticed this a bit too late... see #2044.

File: setup.py [hunk 2]
```diff
@@ -70,7 +69,7 @@
     "learning": ['scikit-learn'],
     "gui-jupyter": ["hyperspy_gui_ipywidgets"],
     "gui-traitsui": ["hyperspy_gui_traitsui"],
-    "mrcz-blosc": ["blosc>=1.5"],
+    "mrcz": ["blosc>=1.5", 'mrcz>=0.3.6'],
```
Comment (ericpre) [inline setup.py]: https://github.com/conda-forge/staged-recipes/pull/5670 just needs a rebuild and then it is ready to merge... but yes for this release this is not going to be ready!

**Comment by francisco-dlp** [comment]
I have to go home, we'll release on Monday.

**Pull request merged**